<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myStatusbar</class>
 <widget class="QWidget" name="myStatusbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>192</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 209, 255);</string>
     </property>
     <property name="text">
      <string>height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="hlineEdit">
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 209, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 209, 255);</string>
     </property>
     <property name="text">
      <string>wdith</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="wlineEdit">
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 209, 255);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 209, 255);</string>
     </property>
     <property name="text">
      <string>penSize</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="pensizeEdit">
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 209, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 209, 255);</string>
     </property>
     <property name="text">
      <string>penStyle</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="StyleBox">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>SolidLine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DashLine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DotLine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DashDotLine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DashDotDotLine</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
